python/src/scipp/table_html/formatting_html.py

```diff
@@ -171,37 +171,69 @@ def _icon(icon_name):
             "</svg>".format(icon_name))
 
 
-def summarize_coord(dim, var):
+def summarize_coord(dim, var, bin_edges=None):
     is_index = dim in var.dims
-    return summarize_variable(str(dim), var, is_index)
+    return summarize_variable(str(dim), var, is_index, bin_edges=bin_edges)
+
+
+def find_bin_edges(var, ds):
+    """
+    Checks if the coordinate contains bin-edges.
+    """
+    bin_edges = []
+    for idx, dim in enumerate(var.dims):
+        len = var.shape[idx]
+        if dim in ds.dims:
+            if ds.shape[ds.dims.index(dim)] + 1 == len:
+                bin_edges.append(dim)
+    return bin_edges
 
 
 def summarize_coords(coords):
     vars_li = "".join(
-        f"<li class='xr-var-item'>{summarize_coord(dim, values)}</span></li>"
-        for dim, values in coords)
+        "<li class='xr-var-item'>"
+        f"{summarize_coord(dim, var, bin_edges=find_bin_edges(var, ds))}"
+        "</span></li>"
+        for dim, var in ds.coords)
 
     return f"<ul class='xr-var-list'>{vars_li}</ul>"
 
 
-def summarize_variable(name, var, is_index=False, has_attrs=False):
+def _make_dim_labels(dim, sparse_dim, bin_edges=None):
+    # Note: the space needs to be here, otherwise
+    # there is a trailing whitespace when no dimension
+    # label has been added
+    if bin_edges and dim in bin_edges:
+        return " [bin-edge]"
+    elif dim == sparse_dim:
+        return " [sparse]"
+    else:
+        return ""
+
+
+def summarize_variable(name, var,
+                       is_index=False,
+                       has_attrs=False,
+                       bin_edges=None):
     """
-    :param name:
-    :param var:
+    :param name: Name of the variable
+    :param var: The variable itself, used to display dtype, unit, values, attrs
     :param is_index: If the variable is an index - used to bold
                      coordinates that represent the indices of
                      a dimension that the data contains
 
     :param has_attrs: If the variable is for a section that cannot contain
                       attributes, then this hides the show/hide button.
+
+    :param is_bin_edges: If true it will add the [bin-edges] variable
     """
     cssclass_idx = " class='xr-has-index'" if is_index else ""
     dims_text = ', '.join(
-        escape(f'{str(dim)} [sparse]' if dim == var.sparse_dim else str(dim))
+        f'{str(dim)}{_make_dim_labels(dim, var.sparse_dim, bin_edges)}'
         for dim in var.dims)
-    dims_str = f"({dims_text})"
+    dims_str = escape(f"{dims_text}".replace("Dim.", ""))
```

Comment:
Saves space, yes, but doesn't it make things more confusing?

Reply:
What about showing () for no dims, and no parenthesis for other dims?
![image](https://user-images.githubusercontent.com/9135965/70995775-4072c480-20c9-11ea-8245-f1ba7cb454f8.png)

Reply:
I think this makes it less obvious for an uninformed user. By having parenthesis *everywhere* it is semi-obvious that *empty parentheses* mean an empty list/tuple.